main.py

```diff
@@ -0,0 +1,203 @@
+import json
+import requests
+import urllib
+import time
+from collections import defaultdict
+import email
+import imaplib
+import sys
+from pony import orm
+import quopri
```

Comment:
It's better to regroup imports: alphabetical order + all `import ...`, then all `from ... import ...`